jsk_unitree_robot/cross/build_ros1.sh

```diff
@@ -32,7 +32,15 @@ mkdir -p ${HOST_INSTALL_ROOT}/ros1_inst
 if [ ${UPDATE_SOURCE_ROOT} -eq 1 ]; then
     vcs import --force --retry 3 --shallow ${SOURCE_ROOT}/src < repos/roseus_no_window.repos
     for dir in euslisp jskeus; do ls ${SOURCE_ROOT}/src/$dir/patches/; rsync -avz ${SOURCE_ROOT}/src/$dir/patches/ ${SOURCE_ROOT}/src/$dir; done
-    sed -i s@:{version}@0.0.0@ ${SOURCE_ROOT}/src/euslisp/package.xml ${SOURCE_ROOT}/src/jskeus/package.xml
+    case ${OSTYPE} in
+        linux*)
+            sed -i s@:{version}@0.0.0@ ${SOURCE_ROOT}/src/euslisp/package.xml ${SOURCE_ROOT}/src/jskeus/package.xml
```

Comment:
linux can use `sed -i'.bak' ` and it seems that latest mac also supports same syntax
https://stackoverflow.com/a/14813278